maintenance-filter-server/src/main/java/org/ccci/maintenance/util/MaintenanceWindowDao.java

```diff
@@ -0,0 +1,18 @@
+package org.ccci.maintenance.util;
+
+import org.ccci.maintenance.MaintenanceWindow;
+import org.joda.time.DateTime;
+
+/**
+ * @author Matt Drees
+ */
+public interface MaintenanceWindowDao
+{
+    MaintenanceWindow getActiveMaintenanceWindow(String filterName, DateTime currentDateTime);
+
+    /** retrieves the window identified by the given id, or null if it does not exist */
+    OwnedMaintenanceWindow getMaintenanceWindowById(String id);
```

Comment:
So, in the past I've hit a few small obstacles from using Guava in the util project. Guava tends to release often, and it adds and removes methods/classes each time -- it doesn't try to stay backwards compatible very much, the way some java libraries try to. And our various apps use different versions of guava -- some old, some new. So trying to make sure the util project uses a version of guava that will 'match' the various guava versions in use has taken a little bit of time. Not much time -- it's not a huge deal or anything. But it's enough of a pain point that I want to be careful of adding a guava dependency here in maintenance-filter. One possibility I would want to evaluate is whether to use JarJar Links or something similar to 'shade' it into the maintenance-filter jar.

So, that's enough that I don't want to do it now, for this PR. I'll create an issue for it, though. Sound good?